samples/mentions.html

```diff
@@ -0,0 +1,257 @@
+								<span class="bullet">•</span>
+								<span class="time">4 hours ago</span>
+								<p class="post-content">
+								Rating has been discontinued because it caused a decline in content quality rather than its increase. People were providing content that would be upvoted, not the content valuable for the community. There are even “secret” groups now!
+								</p>
+							</div>
+						</li>
+					</ul>
+					<div class="editor">
+						<textarea cols="80" id="editor1" name="editor1" rows="10" data-sample="1" data-sample-short>
+							I agree with <a href="mailto: mildred@example.com">@mwilson</a>. Rating system made the quality much worse. In the long run, especially when we have more new members, it will be too hard to control.
+						</textarea>
+					</div>
+					<button id="send" class="send">Send</button>
+				</div>
+
+			</section>
+		</section>
+
+		<footer class="sdk-footer">
+		</footer>
+
+		<script data-sample="1" >
+			var users = [
+				{ id: 1, avatar: 'm_1', fullname: 'Charles Flores', username: 'cflores' },
+				{ id: 2, avatar: 'm_2', fullname: 'Gerald Jackson', username: 'gjackson' },
+				{ id: 3, avatar: 'm_3', fullname: 'Wayne Reed', username: 'wreed' },
+				{ id: 4, avatar: 'm_4', fullname: 'Louis Garcia', username: 'lgarcia' },
+				{ id: 5, avatar: 'm_5', fullname: 'Roy Wilson', username: 'rwilson' },
+				{ id: 6, avatar: 'm_6', fullname: 'Matthew Nelson', username: 'mnelson' },
+				{ id: 7, avatar: 'm_7', fullname: 'Randy Williams', username: 'rwilliams' },
+				{ id: 8, avatar: 'm_8', fullname: 'Albert Johnson', username: 'ajohnson' },
+				{ id: 9, avatar: 'm_9', fullname: 'Steve Roberts', username: 'sroberts' },
+				{ id: 10, avatar: 'm_10', fullname: 'Kevin Evans', username: 'kevans' },
+
+				{ id: 11, avatar: 'w_1', fullname: 'Mildred Wilson', username: 'mwilson' },
+				{ id: 12, avatar: 'w_2', fullname: 'Melissa Nelson', username: 'mnelson' },
+				{ id: 13, avatar: 'w_3', fullname: 'Kathleen Allen', username: 'kallen' },
+				{ id: 14, avatar: 'w_4', fullname: 'Mary Young', username: 'myoung' },
+				{ id: 15, avatar: 'w_5', fullname: 'Ashley Rogers', username: 'arogers' },
+				{ id: 16, avatar: 'w_6', fullname: 'Debra Griffin', username: 'dgriffin' },
+				{ id: 17, avatar: 'w_7', fullname: 'Denise Williams', username: 'dwilliams' },
+				{ id: 18, avatar: 'w_8', fullname: 'Amy James', username: 'ajames' },
+				{ id: 19, avatar: 'w_9', fullname: 'Ruby Anderson', username: 'randerson' },
+				{ id: 20, avatar: 'w_10', fullname: 'Wanda Lee', username: 'wlee' }
+			];
+
+		CKEDITOR.replace( 'editor1', {
+			plugins: 'mentions,emoji,basicstyles,undo,link,wysiwygarea,toolbar',
+			height: 150,
+			toolbar: [
+				{ name: 'document', items: [ 'Undo', 'Redo' ] },
+				{ name: 'basicstyles', items: [ 'Bold', 'Italic', 'Strike' ] },
+				{ name: 'links', items: [ 'Link', 'Unlink' ] }
+			],
+			mentions: [
+				{
+					feed: dataFeed,
+					itemTemplate:
+					'<li data-id="{id}">' +
```

Comment:
This should be indented by a single tab, compared to where the property name starts. I wouldn't mind putting `'<li data-id="{id}">' +` in the same line as the property name TBH.